src/main/java/pl/javagame/App.java

```diff
@@ -7,40 +7,40 @@
 
     public static void main(String[] args) {
 
-        ArrayList<Player> listaGraczy = new ArrayList<>();
+        ArrayList<Player> playerList = new ArrayList<>();
         Scanner scanner = new Scanner(System.in);
-        int iloscGraczy = sprawdzIlloscGraczy();
+        int numberOfPlayers = checkingNoOfPlayers(scanner);
 
         try {
-            for (int i = 1; i < iloscGraczy + 1; i++) {
+            for (int i = 1; i < numberOfPlayers + 1; i++) {
                 System.out.println("Wpisz imię gracza nr: " + i);
-                Player imie = new Player(scanner.nextLine());
-                listaGraczy.add(imie);
+                Player name = new Player(scanner.next());
+                playerList.add(name);
             }
 
         } catch (IllegalArgumentException e) {
             System.out.println("Nie podałeś imienia gracza");
         }
 
         System.out.println("Lista graczy: ");
-        for (Player lista : listaGraczy) {
+        for (Player lista : playerList) {
             System.out.println("Gracz: " + lista);
         }
         scanner.close();
 
     }
 
-    private static int sprawdzIlloscGraczy() {
-        Scanner scanner = new Scanner(System.in);
+    private static int checkingNoOfPlayers(Scanner scanner) {
+        //Scanner scanner = new Scanner(System.in);
```

Comment:
Nie ma sensu zakomentowywać kodu jak masz gita, jeśli będziesz chciał wrócić to zawsze można z historii gita wyciągnąć.